Add event page for Contributor Meetup November 2024 (#1161)
datenanfragen/website: content/en/verein/event/contributor-meetup-11-2024/index.md

{

>     "title": "Contributor Meetup November 2024",

Reviewer: ```suggestion
    "title": "Contributor meetup November 2024",
```

    "type": "event",

>     "event_date": "2024-11-01T17:00:00+02:00",

Reviewer: ```suggestion
    "event_date": "2024-11-06T17:00:00+02:00",
```

    "duration": "120m",
    "place": {
        "online": "https://backend.datenanfragen.de/meet/join/en/contributor-meetup-11-2024"
    },
    "featured_image": "contributor-meetup-november-2024.jpg",
    "description": "A meetup for everybody already working on datarequests.org or wanting to, to come together, make plans and learn new things."
}

> Regardless of whether you are already contributing to datarequests.org, whether you are interested in joining, or you just want to get to know us: Our contributor meetup is supposed to be a space to ask question on contributing, for learning on how you can participate, and where data protection nerds can network on various issues. To take part, you need nothing in particular. Just be in our online meeting on November 7 at 17:00 (CEST) and bring your curiosity, questions and motivation.

Reviewer: ```suggestion
Regardless of whether you are already contributing to datarequests.org, whether you are interested in joining, or you just want to get to know us: Our contributor meetup is supposed to be a space to ask questions on participating, for learning on how you can contribute to datarequests.org and Tweasel, and where data protection nerds can network on various issues. To take part, you need nothing in particular. Just be in our online meeting on November 6 at 17:00 (CEST) and bring your curiosity, questions and motivation.
```

> If you know that you will definitely come, we would appreciate a message to kontakt@datenanfragen.de for our planning.

Reviewer: ```suggestion
If you know that you will definitely come, we would appreciate a message to kontakt@datenanfragen.de for our planning (optional).
```
